fjord/feedback/views.py

```diff
@@ -45,7 +46,17 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
+    opinion_id = request.session['opinion_id']
```

Comment:
If opinion_id isn't in the session, I think this kicks up a `KeyError`. We should handle that scenario.